lapack.go

```diff
@@ -136,3 +136,23 @@ const (
 	InitZ   Comp = 'I'
 	UpdateZ Comp = 'V'
 )
+
+// EigVecSide specifies what eigenvectors will be computed.
+type EigVecSide byte
```

Comment:
We've in the past just modified the blas type to help lapack. I don't think blas actually uses uplo.All. I'm okay with such a modification again.